src/Typing.js

```diff
@@ -26,7 +26,7 @@ class Typing extends Component {
       JSON.stringify(children, getCircularReplacer()) !==
         JSON.stringify(this.props.children, getCircularReplacer())
     ) {
-      this.resetState();
+      // this.resetState();
```

Comment:
Removing this line would remove the ability to update the children of the Typing component after the animation has started. Is commenting out this line necessary to fix this bug?

Reply:
It was when I was messing with it. The text would disappear after they typing animation was complete if this line was placed in. I didn't look into _why_ exactly, but I can look into it some more and see if we can fix it another way.

Updating the children after it's started typing does seem to contradict the functionality of being able to backspace over text though right? If a person backspaced over text, then updated the children of the Typing component, the Typing component would then see the backspaced text as whole again and render it entirely, correct? That's just my initial assumption without ever testing that use case. If I'm wrong then great!

Reply:
Yes, I think that is why the entire state was being reset any time the children were changed. That way, the animation would just restart from scratch if the children were ever changed, and the state would be reset to as if the component had just been created/mounted with the new children.